src/main.c

```diff
@@ -1435,14 +1435,14 @@ static int MakeConsole() {
     if (!ok) {
         if (!AttachConsole(ATTACH_PARENT_PROCESS)) {
             FreeConsole();
-            if (!AllocConsole()) {
-                err = GetLastError();
-                if (err==ERROR_ACCESS_DENIED) {
-                    MessageBox(NULL, "Insufficient rights to create a console window", "Error", MB_ICONERROR);
-                }
-            }
-            AttachConsole(GetCurrentProcessId());
-        }
+            if (!AllocConsole()) {
```

Comment:
yes, I think the ide converted tabs to spaces or something like that.
I'll revert ..